drivers/storage/portworx/component/telemetry.go

```diff
@@ -1004,41 +1008,59 @@ func getArcusRegisterProxyURL(cluster *corev1.StorageCluster) string {
 }
 
 // CanAccessArcusRegisterEndpoint checks if telemetry registration endpoint is reachable
-func CanAccessArcusRegisterEndpoint(cluster *corev1.StorageCluster) (bool, error) {
+// return true immediately if it can reach to Arcus
+// return false after failing 5 times in a row
+func CanAccessArcusRegisterEndpoint(
```

Comment:
Let's do some functional tests for this API. you may refer to this function for tests

func TestRemoteManifestWithProxy(t *testing.T) {